app/static/settings/translations.json

```diff
@@ -554,27 +554,27 @@
     "lang_zh-TW": {
         "search": "搜尋",
         "config": "設定",
-        "config-country": "設置國家",
-        "config-lang": "界面語言",
+        "config-country": "設定國家",
+        "config-lang": "介面語言",
         "config-lang-search": "搜尋語言",
         "config-near": "接近",
-        "config-near-help": "城市名",
-        "config-block": "排除",
-        "config-block-help": "網址列表，以逗號分隔",
-        "config-block-title": "按標題屏蔽",
-        "config-block-title-help": "使用正則表達式",
-        "config-block-url": "按網址屏蔽",
-        "config-block-url-help": "使用正則表達式",
+        "config-near-help": "城市名稱",
+        "config-block": "封鎖",
+        "config-block-help": "以逗號分隔之網址列表",
+        "config-block-title": "按標題封鎖",
+        "config-block-title-help": "使用正規表達式",
+        "config-block-url": "按網址封鎖",
+        "config-block-url-help": "使用正規表達式",
```

Comment:
I would keep using "排除" instead of "封鎖" in these sentences sense that these are just removing web sites from searchresults, instead of making them non-visitble.

That said, such distinction might never be needed in whoogle and those two terms would be equally good.